servers/src/test/java/tachyon/worker/block/BlockMetadataManagerTest.java

```diff
@@ -52,18 +54,21 @@
 
   private BlockMetadataManager mMetaManager;
 
-  @Rule
-  public TemporaryFolder mFolder = new TemporaryFolder();
+  @ClassRule
+  public static TemporaryFolder sFolder = new TemporaryFolder();
 
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
 
-  @Before
-  public void before() throws Exception {
-    String baseDir = mFolder.newFolder().getAbsolutePath();
+  @BeforeClass
+  public static void setupTieredStorage() throws Exception {
```

Comment:
The convention for methods annotated with `@BeforeClass` is to name them `beforeClass`